setup.py

```diff
@@ -31,7 +31,7 @@
 if py_version < '2.6':
   REQUIRED_PACKAGES.append('simplejson')
 
-_PROTORPC_VERSION = '0.9.1'
+_PROTORPC_VERSION = '0.9.2a1'
```

Comment:
let's just call it `0.9.2`? we don't have a particularly formal release process.